webviz_subsurface/plugins/_multiple_regression.py

```diff
@@ -0,0 +1,1129 @@
+                for ens in ensembles
+            }
+            self.parameterdf = load_parameters(
+                ensemble_paths=self.ens_paths, ensemble_set_name="EnsembleSet"
+            )
+            if self.response_file:
+                self.responsedf = load_csv(
+                    ensemble_paths=self.ens_paths,
+                    csv_file=response_file,
+                    ensemble_set_name="EnsembleSet",
+                )
+            else:
+                self.responsedf = load_smry(
+                    ensemble_paths=self.ens_paths,
+                    column_keys=self.column_keys,
+                    time_index=self.time_index,
+                )
+                self.response_filters["DATE"] = "single"
+        else:
+            raise ValueError(
+                'Incorrect arguments.\
+                 Either provide "csv files" or "ensembles and response_file".'
+            )
+        self.check_runs()
+        self.check_response_filters()
+        if response_ignore:
+            self.responsedf.drop(response_ignore, errors="ignore", axis=1, inplace=True)
+        if response_include:
+            self.responsedf.drop(
+                self.responsedf.columns.difference(
+                    ["REAL", "ENSEMBLE", *response_include, *list(response_filters.keys()),]
+                ),
+                errors="ignore",
+                axis=1,
+                inplace=True,
+            )
+        if parameter_ignore:
+            self.parameterdf.drop(parameter_ignore, axis=1, inplace=True)
+
+        self.theme = app.webviz_settings["theme"]
+        self.badge_style = {
+            "font-weight": "bold",
+            "fontSize": ".70em",
+            "borderRadius": "3px",
+            "backgroundColor": "#505050",
+            "color": "white",
+            "white-space": "pre-wrap",
+        }
+        self.tooltip_style = {
+            "fontSize": ".75em",
+            "backgroundColor": "#505050",
+            "color": "white",
+            "opacity": "85%",
+            "white-space": "pre-wrap",
+            "borderRadius": "5px",
+        }
+        self.set_callbacks(app)
+
+
+    @property
+    def tour_steps(self):
+        """ Adding a "Guided tour" functionality """
+        steps = [
+            {
+                "id": self.uuid("layout"),
+                "content": (
+                    "Dashboard displaying the results of a multiple "
+                    "regression of input parameters and a chosen response."
+                ),
+            },
+            {
+                "id": self.uuid("p-values-plot"),
+                "content": (
+                    "A plot showing the p-values for the parameters from the table ranked from most "
+                    "significant to least significant.  Red indicates that the p-value is "
+                    "significant, gray indicates that the p-value is not significant."
```

Comment:
Should be careful and not claim significance. Red is also a theme color in your case, when not using the Equinor theme, this description is invalid.
```suggestion
                    "significant to least significant (low to high). Bars are highlighted when the p-values are "
                    "less than 0.05, meaning that the terms are likely to be significant. Otherwise the bars are colored gray."
```